github/projects.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ProjectsService handles communication with the project V2
+// methods of the GitHub API.
+//
+// GitHub API docs: https://docs.github.com/rest/projects/projects
+type ProjectsService service
+
+func (p ProjectV2) String() string { return Stringify(p) }
```

Comment:
I think we should move `ProjectV2` structure here from https://github.com/google/go-github/blob/132df79c37d51098eec0a2d9250e8bff65e46293/github/event_types.go#L1114-L1141